### ISSUE TITLE
The app expects 'public' to be writable but documentation does not seem to reflect it

### ISSUE THREAD

**Issue by C0rn3j** [freescout-helpdesk/freescout#69]
Had to run 
chown -R www-data:www-data public

to get the app to load properly.

**Comment by freescout-helpdesk** [comment]
Yes. Better to do chown for the whole application directory as mentioned here: https://github.com/freescout-helpdesk/freescout/wiki/Installation-Guide#5-installing-the-application

**Comment by C0rn3j** [comment]
That chowns the /var/www/html folder only and the subsequent git clone is executed as the running user (which in my case was root).

You'd have to git clone as the newly created user for that to take effect, and that is not mentioned in the documentation.
